templates/zerver/api/incoming-webhooks-walkthrough.md

```diff
@@ -36,6 +36,25 @@ thing, it requires only one fixture,
 }
 ```
 
+### HTTP Headers
+
+If your integration depends on HTTP headers sent by the service you are
+integrating with, then it would be a good idea to store these too. An
+example of when you would need to do this would be if the service sends
+the event type as a custom http header, like with GitHub's webhooks.
+If your integration does not depend on HTTP headers, you don't need to
+follow these additional steps.
+
+To do this,
+
+1. In the folder where your integration's code lives, create a file callled
+`headers.py`.
+
+2. Inside this file, create a method with the name `fixture_to_headers`. This
+method should take a fixture's name as it's first argument and return a
+dictionary of headers that you'd like to associate the fixture with. For
+example, it should return something like `{"X-Event_Type": "ping"}`.
```

Comment:
We should add a section to this guide that recommends our new naming convention for fixtures! :)

Reply:
I did a bit more tweaking of these docs, and also moved them below the next paragraph, which this had inadvertently moved inside this "HTTP headers" section.

As a sidenote, @Hypro999 you should setup a spellchecker in your editor; I had to fix the  "callled" above, and I feel like I've had to fix typos a lot in your PRs.

Reply:
Ah, sorry about that, I just went ahead and enabled a spellchecker.